subprojects/docs/src/docs/userguide/jvm/java_testing.adoc

```diff
@@ -93,6 +93,8 @@ You can also enable this behavior by using the `--fail-fast` command line option
 `testLogging` — default: _not set_::
 This property represents a set of options that control which test events are logged and at what level. You can also configure other logging behavior via this property. See link:{javadocPath}/org/gradle/api/tasks/testing/logging/TestLoggingContainer.html[TestLoggingContainer] for more detail.
 
+Also, you are able to dry run your tests with `--test-dry-run` CLI option. When this option is used, Gradle will simulate the execution of the tests without actually running them. Instead, Gradle will generate a report that lists all the tests that would have been executed based on your filtering configuration. The `--test-dry-run` option can be useful when you want to verify that your test filtering configuration is correct without actually running the tests.
```

Comment:
This should be formatted like the other parameters, since it is still a normal property as well:
```suggestion
`dryRun` — default: false::
If this property is `true`, Gradle will simulate the execution of the tests without actually running them. This will still generate reports, allowing for inspection of what tests were selected. This can be used to verify that your test filtering configuration is correct without actually running the tests.

You can also enable this behavior by using the `--test-dry-run` command-line option, or disable it respectively with `--no-test-dry-run`.
```